tests/test_attributions_llama.py

```diff
@@ -200,23 +202,19 @@ def verify_llama_3_2_1b(s: str):
     verify_feature_edges(model, graph)
 
 
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="CUDA not available")
 def test_small_llama_model():
     s = torch.tensor([10, 3, 4, 3, 2, 5, 3, 8])
     verify_small_llama_model(s)
 
 
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="CUDA not available")
 def test_large_llama_model():
     s = torch.tensor([0, 113, 24, 53, 27])
     verify_large_llama_model(s)
 
 
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="CUDA not available")
 def test_llama_3_2_1b():
     s = "The National Digital Analytics Group (ND"
     verify_llama_3_2_1b(s)
-
-
-if __name__ == "__main__":
```

Comment:
I'm in favor of keeping these entrypoints, less friction for debugging

Reply:
You can run any specific test file with pytest directly using `pytest test/test_attributions_llama.py`. Is there an extra advantage to keeping the main given this functionality is currently provided by pytest? Using a main function this would be run with `python test/test_attributions_llama.py`, which seems like an equivalent command. You can also run individual tests by regex match, e.g. `pytest -k small_llama` which is even less typing and allows more targeting.

Reply:
I am aware about that, was referring to debugging the test without setting up the tests in vs code. I don't care much so feel free to remove if this feels important

Reply:
You don't have to set up anything in vscode, I usually run pytest from CLI personally so it would be the same as running manually in CLI with a `__main__`